src/clr-angular/data/datagrid/datagrid-column-toggle.ts

```diff
@@ -22,6 +22,7 @@ import { DatagridColumnChanges } from './enums/column-changes.enum';
       #anchor
       (click)="toggleSwitchPanel()"
       class="btn btn-sm btn-link column-toggle--action"
+      aria-controls="expandable" 
```

Comment:
I believe you should be assigning a dynamic id to the div that gets toggled by this button, and set the `aria-controls` value to that id.